doc/tutorials/add_solver.rst

```diff
@@ -234,24 +234,21 @@ Specifying the solver parameters
 If your solver has hyperparameters, you can specify them by adding an attribute ``parameters``.
 This attribute is a dictionary whose keys are the solver's hyperparameters.
 
-For example, if our solver has two hyperparameters, ``stepsize`` and ``momentum``, we implement them as follows:
+For our ridge estimator, we can set the two hyperparameters ``positive`` and ``solver`` as follows
 
 .. code-block:: python
-    :caption: benchmark_ridge/solvers/mysolver.py
+    :caption: benchmark_ridge/solvers/sklearn.py
```

Comment:
I'm not convinced: the sklearn file already exists in the benchmark. I'd keep mysolver.